src/openrct2-ui/windows/Map.cpp

```diff
@@ -34,15 +34,21 @@
 #include <vector>
 
 #define MAP_COLOUR_2(colourA, colourB) (((colourA) << 8) | (colourB))
-#define MAP_COLOUR(colour) MAP_COLOUR_2(colour, colour)
-#define MAP_COLOUR_UNOWNED(colour) (PALETTE_INDEX_10 | ((colour)&0xFF00))
+#define MAP_COLOUR(flashColour) MAP_COLOUR_2(flashColour, flashColour)
+#define MAP_COLOUR_UNOWNED(flashColour) (PALETTE_INDEX_10 | ((flashColour)&0xFF00))
```

Comment:
Those are good candidates to be converted to constexpr functions I think.

Reply:
urgh didn't mean to rename these